src/views/infra/network/EditTrafficLabel.vue

```diff
@@ -0,0 +1,241 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+<template>
+  <div class="form-layout">
+    <a-spin :spinning="loading">
+      <a-form :form="form" :loading="loading" @submit="handleSubmit" layout="vertical">
+        <a-form-item>
+          <span slot="label">
+            {{ $t('label.id') }}
+            <a-tooltip :title="apiParams.id.description">
+              <a-icon type="info-circle" style="color: rgba(0,0,0,.45)" />
+            </a-tooltip>
+          </span>
+          <a-select
+            v-decorator="['id', {
+              initialValue: selectedType,
+              rules: [{ required: true, message: $t('message.error.select') }] }]"
+            :loading="typeLoading"
+            :placeholder="apiParams.id.description"
+            @change="onChangeTrafficType">
+            <a-select-option v-for="type in trafficTypes" :key="type.id">
+              {{ type.traffictype }}
+            </a-select-option>
+          </a-select>
+        </a-form-item>
+        <a-form-item>
+          <span slot="label">
+            {{ $t('label.kvmnetworklabel') }}
+            <a-tooltip :title="apiParams.kvmnetworklabel.description">
+              <a-icon type="info-circle" style="color: rgba(0,0,0,.45)" />
+            </a-tooltip>
+          </span>
+          <a-input
+            v-decorator="['kvmnetworklabel', {
+              initialValue: trafficResource.kvmnetworklabel || $t('label.network.label.display.for.blank.value')
```

Comment:
@utchoang Would it be better to use `$t('label.network.label.display.for.blank.value')` as a placeholder instead?
Also the user is unable to clear or reset the tag once it's been set